test/order_test.rb

```diff
@@ -1,9 +1,9 @@
-require 'minitest/autorun'
-require 'minitest/reporters'
-require 'minitest/skip_dsl'
+require "minitest/autorun"
+require "minitest/reporters"
+require minitest/skip_dsl"
```

Comment:
Syntax error missing `"`.

```suggestion
require "minitest/skip_dsl"
```